packages/target-tauri/src-tauri/src/webxdc/commands_main_window.rs

```diff
@@ -231,8 +232,48 @@ pub(crate) async fn open_webxdc<'a>(
     message_id: u32,
     href: String,
 ) -> Result<(), Error> {
-    let uuid = Uuid::new_v4().to_string();
-    let window_id = format!("webxdc:{uuid}");
+    let window_id: String = {
+        let get_random_chars = || rand::distr::Alphanumeric.sample_string(&mut rand::rng(), 64);
+        let webxdc_window_state_store_name = ".webxdc-window-state-secret.json";
+        // Why do we need to store a secret just for making a window label?
+        // Because we want to keep the window label fixed
+        // per webxdc app message,
+        // so that the window state (position, size) is stored
+        // using `tauri_plugin_window_state`,
+        // but at the same time we don't want to expose
+        // the account_id and message_id to the webxdc app
+        // (through `globalThis.__TAURI_INTERNALS__.metadata.currentWindow.label`)
+        // because account_id and message_id can give away how many accounts
+        // and messages the user has.
+        // So the app could
+        //
+        // TODO we probably need to consider a more secure way
+        // to store this secret. But it's not a super important one.
```

Comment:
The primary attack we worry about here is that the webxdc gets it's window label/id and thus an attacker in the same chat could infer how many messages and accounts you have.

Reply:
But the other thing here could be missing housekeeping, if you delete a webxdc, it's id and position still stay in the window size config file, which is wasting some space, not notable amounts, but still information is retained that is not needed anymore.

Reply:
> The primary attack we worry about here is that the webxdc gets it's window label/id and thus an attacker in the same chat could infer how many messages and accounts you have.

Well yes, that is solved, but still, it's a secret, and I don't know much about storing them securely. E.g. see the "local database encryption" conversation.